product_profile/views/config_view.xml

```diff
@@ -5,32 +5,22 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="sale.res_config_settings_view_form" />
         <field name="arch" type="xml">
-            <xpath
-                expr="//div[@data-key='sale_management']//div[hasclass('o_settings_container')]"
-                position="inside"
-            >
-                <div class="col-12 col-lg-6 o_setting_box">
-                    <div class="o_setting_left_pane">
+            <xpath expr="//block[@name='catalog_setting_container']" position="inside">
+                    <setting>
                         <field name="group_product_profile" />
-                    </div>
-                    <div class="o_setting_right_pane">
-                        <label for="group_product_profile" />
                         <div
-                            class="content-group"
-                            groups="product_profile.group_product_profile_manager"
-                        >
-                            <div class="mt16">
-                                <button
-                                    name="%(product_profile.product_profile_action)d"
-                                    icon="fa-arrow-right"
-                                    type="action"
-                                    string="Product Profiles"
-                                    class="btn-link"
-                                />
-                            </div>
+                        class="content-group"
+                        groups="product_profile.group_product_profile_manager"
+                    >
+                            <button
+                            name="%(product_profile.product_profile_action)d"
+                            icon="fa-arrow-right"
+                            type="action"
+                            string="Product Profiles"
+                            class="btn-link"
+                        />
```

Comment:
This indent is a bit messed up, it alter the readability as `<button` has more spaces that it's closing `/>`

Reply:
I've manually edited the indentation. Then run pre-commit and forced pushed. Should be ok now.